### PR TITLE
[wasm] Set runtime pack directory to the nuget path, if not set

### PR DESCRIPTION
**Pull request by radical**

### CODE REVIEW

File: src/mono/wasm/build/WasmApp.targets [hunk 1]
```diff
@@ -199,6 +199,11 @@
   </Target>
 
   <Target Name="_BeforeWasmBuildApp" DependsOnTargets="_SetupEmscripten;_SetWasmBuildNativeDefaults">
+    <PropertyGroup>
+      <MicrosoftNetCoreAppRuntimePackDir Condition="'$(MicrosoftNetCoreAppRuntimePackDir)' == ''">$([MSBuild]::NormalizeDirectory($(NuGetPackageRoot), 'microsoft.netcore.app.runtime.browser-wasm', $(BundledNETCoreAppPackageVersion)'))</MicrosoftNetCoreAppRuntimePackDir>
```
Comment (lewing) [inline src/mono/wasm/build/WasmApp.targets:203]:
```suggestion
      <MicrosoftNetCoreAppRuntimePackDir Condition="'$(MicrosoftNetCoreAppRuntimePackDir)' == ''">$([MSBuild]::NormalizeDirectory($(NuGetPackageRoot), 'microsoft.netcore.app.runtime.mono.browser-wasm', $(BundledNETCoreAppPackageVersion)'))</MicrosoftNetCoreAppRuntimePackDir>
```

**Review by lewing — CHANGES_REQUESTED** [review]

File: src/mono/wasm/build/WasmApp.targets (hunk 1 above)
Comment (radical) [reply src/mono/wasm/build/WasmApp.targets:203]: the name changed only for the package id

Comment (lewing) [reply src/mono/wasm/build/WasmApp.targets:203]: https://github.com/dotnet/sdk/blob/main/src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.Current.targets#L75

Comment (radical) [reply src/mono/wasm/build/WasmApp.targets:203]: Lol my bad! We are using that nuget package path, so that would be the new name!

**Review by lewing — APPROVED** [review]

**Pull request merged**